### PR TITLE
PRM-944 Create connector to new MS from backend

### PR DESCRIPTION
**Pull request by ruthdas03**

### CODE REVIEW

File: it/connectors/FileNotificationOrchestratorConnectorISpec.scala [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+class FileNotificationOrchestratorConnectorISpec extends IntegrationSpecCommonBase {
+
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
+
+  class Setup {
+    val connector: FileNotificationOrchestratorConnector = injector.instanceOf[FileNotificationOrchestratorConnector]
+  }
+
+  "getFileNotificationOrchestrator" should {
```
Comment (AndrewRButcher) [inline it/connectors/FileNotificationOrchestratorConnectorISpec.scala:34]: Tests need changing to match post request

File: app/config/AppConfig.scala [hunk 2]
```diff
@@ -41,6 +41,8 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   lazy val desBearerToken: String = s"Bearer ${servicesConfig.getConfString("des.bearerToken", "")}"
 
+  lazy val fileNotificationOrchestrator: String = servicesConfig.baseUrl("penalties-file-notification-orchestrator")
```
Comment (AndrewRButcher) [inline app/config/AppConfig.scala:44]:
```suggestion
  lazy val fileNotificationOrchestrator: String = servicesConfig.baseUrl("penalties-file-notification-orchestrator")

  def postFileNotificationUrl: String = s"$fileNotificationOrchestrator/penalties-file-notification-orchestrator/new-notifications"```

File: it/utils/FileNotificationOrchestratorWiremock.scala [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+package utils
+
+import com.github.tomakehurst.wiremock.client.WireMock.{aResponse, post, stubFor, urlEqualTo}
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+
+trait FileNotificationOrchestratorWiremock {
+
+  def mockResponseForFileNotificationOrchestrator(status: Int): StubMapping = {
+    stubFor(post(urlEqualTo(s"")) //TODO update url
```
Comment (AndrewRButcher) [inline it/utils/FileNotificationOrchestratorWiremock.scala:25]:
```suggestion
    stubFor(post(urlEqualTo(s"/penalties-file-notification-orchestrator/new-notifications"))
```

**Review by AndrewRButcher — COMMENTED** [review]
Added few suggestions to begin with see below

- [ ]  Change GET to POST request
- [ ] Update Url
- [ ] Add tests
- [ ] Also don't forget to add a Unit test Spec for the new connector

File: conf/application.conf [hunk 4]
```diff
@@ -138,6 +138,11 @@ microservice {
       environment = "ist0"
       bearerToken = "1234"
     }
+
+    penalties-file-notification-orchestrator {
+      host = localhost
+      port = 9184
+    }
```
Comment (AndrewRButcher) [inline conf/application.conf:145]: Can we have an app-config base PR to override this in the environments?

**Review by AndrewRButcher — APPROVED** [review]

**Comment by BackRowOla** [comment]
[App-config-base PR](https://github.com/hmrc/app-config-base/pull/6603)

**Pull request merged**